demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -285,16 +284,27 @@ def convert_contribution_to_pack(self, files_to_source_mapping: Dict = None):
                 basename = os.path.basename(pack_subdir)
                 if basename in {SCRIPTS_DIR, INTEGRATIONS_DIR}:
                     self.content_item_to_package_format(
-                        pack_subdir, del_unified=True, source_mapping=files_to_source_mapping
+                        pack_subdir, del_unified=False, source_mapping=files_to_source_mapping
                     )
 
+                elif basename == 'Playbooks':
+                    files = get_child_files(pack_subdir)
+                    for file in files:
+                        file_name = os.path.basename(file)
+                        if file_name.endswith('.yml'):
+                            self.generate_readme_for_converted_pack(file)
```

Comment:
Let's adjust the function name from `generate_readme_for_converted_pack` to something more appropro like `generate_readme_for_pack_content_item` or `generate_readme_for_pack_content_entity`.